### PR TITLE
Amendment: Make the declaration not part of the constitution

### PR DESCRIPTION
**Pull request by JMAlego**
### Rationale

The Constitution should be a static document without a requirement to change it every time someone gets elected. This proposal keeps the requirement for a declaration without requiring changes to the Constitution every time someone gets elected.

### CODE REVIEW

**Comment by JMAlego** [comment]
### Notes

YUSU may not like it but, as with all amendments, they get a veto on all amendments anyway.

**Comment by JMAlego** [comment]
### Vote

**Final vote:** 23 for (100%), 0 against, 0 abstaining
**Passed**

**Comment by JMAlego** [comment]
Changes in 5feea57ac81fb405d5bb4f25b7a8acfa59c9d3f5 are up for discussion as they are caused by the amendment but not specified by the amendment.

File: Declaration.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# HackSoc Committee Declaration
+
+*This document is to be signed by all members of the incoming Committee before they take office. The signed Declaration should be made public on the HackSoc website by the incoming Committee once signed.*
```
Comment (sersorrel) [inline Declaration.md]: idk words are hard but maybe something like this

```suggestion
*This template document is to be signed by all members of the incoming Committee before they take office. The signed Declaration should be made public on the HackSoc website by the incoming Committee once signed (the signed copy should not be committed here).*
```

**Pull request merged**